iree/vm/ops.h

```diff
@@ -87,6 +90,7 @@ static inline float vm_div_f32(float lhs, float rhs) { return lhs / rhs; }
 static inline float vm_rem_f32(float lhs, float rhs) {
   return remainderf(lhs, rhs);
 }
+static inline float vm_fma_f32(float a, float b, float c) { return a * b + c; }
```

Comment:
would the code suggested here be useful? https://www.cplusplus.com/reference/cmath/fma/

```c
#ifdef FP_FAST_FMA
  result = fma(x,y,z);
#else
  result = x*y+z;
#endif
```

Reply:
Good call - done!